agreed/0009-MPID-Messaging/0009-MPID-Messaging.md

```diff
@@ -381,7 +371,7 @@ impl MpidMessage {
                                                      public_key) && self.header.verify(public_key)
     }
 
-    fn encode(recipient: &::NameType, body: &Vec<u8>) -> Vec<u8> {
+    fn encode(recipient: &XorName, body: &Vec<u8>) -> Vec<u8> {
         ::utils::encode(&(recipient, body)).unwrap_or(vec![])
```

Comment:
I think `serialise`/`deserialise` is the preferred way now, but I don't think we need the wrapper.  The `encode` here is just to allow us to create serialised data in order to sign it - it's probably OK as shown here (just create a tuple on the fly).